src/Form/UserPickupRequestForm.php

```diff
@@ -395,7 +402,7 @@ public function arborcat_mail($key, $email_to, $patron, $code, $holds) {
       foreach ($holds as $title) {
         $email_message = $email_message.$title['Title']."\r\n";
       }
-      $mailManager = \Drupal::service('plugin.manager.mail');
+      $mail_manager = \Drupal::service('plugin.manager.mail');
```

Comment:
Again, looks like we are instantiating the drupal mail service but then just using the PHP mail() function on the next line. Can probably just delete this line.